doonut-app-external-api/src/main/kotlin/com/doonutmate/calendar/controller/CalendarController.kt

```diff
@@ -9,15 +11,16 @@ import com.doonutmate.oauth.configuration.Authorization
 import io.swagger.v3.oas.annotations.Operation
 import io.swagger.v3.oas.annotations.Parameter
 import io.swagger.v3.oas.annotations.tags.Tag
+import jakarta.validation.Valid
 import org.springframework.http.ResponseEntity
 import org.springframework.web.bind.annotation.GetMapping
 import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.PutMapping
 import org.springframework.web.bind.annotation.RequestBody
 import org.springframework.web.bind.annotation.RequestMapping
 import org.springframework.web.bind.annotation.RequestParam
 import org.springframework.web.bind.annotation.RestController
 import java.time.Instant
-
 @RestController
```

Comment:
import문과 본문 사이에 줄바꿈이 필요할 것 같아요.
```suggestion

@RestController
```